lib/console.js

```diff
@@ -35,20 +37,34 @@ function Console(stdout, stderr) {
   }
 }
 
+function write(ignoreErrors, stream, string) {
+  if (!ignoreErrors) return stream.write(string);
+
+  try {
+    stream.write(string, (err) => {
+      if (err) {
+        stream.once('error', () => {});
+      }
+    });
```

Comment:
@mcollina This is `.once`, not `.on`? If the callback sees an `err` argument, then that error *will* be emitted as an `'error'` event, so the extra listener *will* be called and removed.